hangman.py

```diff
@@ -152,9 +228,16 @@ def show_possible_matches(my_word):
              that has already been revealed.
 
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
+    my_word_new = my_word.replace(' ', '')
+    possible_matches = []
+    for word in wordlist:
+        if match_with_gaps(my_word_new, word):
+            possible_matches.append(word)
 
+    if possible_matches is []:
+        print('No matches founded')
+    else:
+        print(' '.join(possible_matches))
 
 
 def hangman_with_hints(secret_word):
```

Comment:
This function is mostly the same as `hangman`. Could you avoid code duplication and combine them?